README.md

```diff
@@ -24,19 +24,32 @@ Plug 'zchee/deoplete-go', { 'do': 'make'}
 
 ## Settings
 
-### Class sorting
-By Default, completion list is the sort order of gocode.  
-If you want to change to the arbitrary sort order, set the order in `deoplete#sources#go#sort_class`.  
+### Class Aligning
+By default the classes (var, func, type, const, package) are not aligned in the popup menu.
 
-Available values is `package`, `func`, `type`, `var`, `const`.  
-`deoplete#sources#go#sort_class` **value must be include all class.**
+First is unaligned, second is aligned.
+
+<img src="https://raw.githubusercontent.com/zchee/deoplete-go/master/images/align_class.png" width="600">
+
+If you would like them aligned just set:
+
+```vim
+let g:deoplete#sources#go#align_class = 1
```

Comment:
Alright seems like the alignment is only for the keywords. So this gets a +1 from me then.